grpc/src/test/java/com/linecorp/armeria/server/grpc/UnframedGrpcErrorHandlerTest.java

```diff
@@ -122,4 +165,21 @@ void jsonWithStackTrace() {
                                        "\"stack-trace\":\"io.grpc.StatusException");
         assertThat(response.trailers()).isEmpty();
     }
+
+    @Test
+    void richJson() throws JsonProcessingException {
+        final BlockingWebClient client = testServerGrpcStatus.webClient().blocking();
+        final AggregatedHttpResponse response =
+                client.prepare()
+                      .post(TestServiceGrpc.getEmptyCallMethod().getFullMethodName())
+                      .content(MediaType.PROTOBUF, Empty.getDefaultInstance().toByteArray())
+                      .execute();
```

Comment:
A response can be directly converted into `JsonNode` using:
```suggestion
        final ResponseEntity<JsonNode> response =
                client.prepare()
                      .post(TestServiceGrpc.getEmptyCallMethod().getFullMethodName())
                      .content(MediaType.PROTOBUF, Empty.getDefaultInstance().toByteArray())
                      .asJson(JsonNode.class)
                      .execute();
```

Reply:
It seems like an exception would be thrown in `AggregatedResponseAs.newJsonResponseEntity()` when the status of the response is not SUCCESS 🙏

Reply:
I see. Let's keep it as it is.